taxcalc/tests/test_parameters.py

```diff
@@ -210,75 +212,40 @@ def test_json_file_contents(tests_path, fname):
                               param.get('indexed', False),
                               param.get('indexable', False))
             failures += fail + '\n'
-        # check that value_type is correct string
```

Comment:
Does this need to be ported over?

Reply:
This just checks that `value_type` is one of `['boolean', 'integer', 'real', 'string']`. `value_type` was removed in favor of `type`. ParamTools uses `marshmallow` to validate that only correct types are used: https://github.com/PSLmodels/ParamTools/blob/master/paramtools/schema.py#L100

Reply:
No I don't think so.